### ISSUE TITLE
Randomly repair options don't show

### ISSUE THREAD

**Issue by jonpas** [KoffeinFlummi/AGM#2004]
I can't find good reproduction steps for this, it just happens randomly.

What happens is when you Check Vehicle not all options for Repair show up, most notably the Tail Rotor (because I repair it the most probably), but then after walking around doing some more checks it'll come up eventually.

**Comment by jonpas** [comment]
Just happened to me on Vanilla Ghosthawk, but I couldn't repair either Main Rotor nor Tail Rotor, both showed up fine in the Check Vehicle hint and afterwards without changing anything I could normally repair Pawnee's Tail Rotor.

**Comment by commy2** [comment]
toolkits don't get used up. are you doing this on the server? Arma may take a while to sync hit point damage on non-local vehicles.

**Comment by jonpas** [comment]
Yep testing on local dedicated (hosted on same machine), same time as I am testing ATRQ repair bug.
But if you go into that vehicle that vehicle becomes local to you, not sure what happens when you get out though.

What do you mean they don't get used up, does that mean one Toolkit can be used for infinite repairs? I am missing something here. <.<

**Comment by commy2** [comment]
`does that mean one Toolkit can be used for infinite repairs`

yes.

**Comment by jonpas** [comment]
Oh wow, I must be stupid then.
